pkgs/by-name/si/simulide/package.nix

```diff
@@ -0,0 +1,99 @@
+{ lib
+, stdenv
+, fetchbzr
+, substituteAll
+, libsForQt5
+, old ? false
+}:
+
+let
+  versionTable = {
+    old = {
+      version = "0.4.15";
+      release = "SR10";
+      branch = "simulide_0.4.14"; # the branch name does not mach the version for some reason
+      rev = "291";
+      sha256 = "sha256-BBoZr/S2pif0Jft5wrem8y00dXl08jq3kFiIUtOr3LM=";
+    };
+    new = {
+      version = "1.1.0";
+      release = "RC0";
+      branch = "1.1.0";
+      rev = "1909";
+      sha256 = "sha256-AvucduVraSlTXT8Hnpgh9i5Tsxji++vgjAJX1ZFnMs4=";
+    };
+  };
+  versionInfo = if old then versionTable.old else versionTable.new;
```

Comment:
It would be preferrable to keep a single version. As said before, rather create a generic mechanism to handle multiple version later.